sympy/stats/crv_types.py

```diff
@@ -2627,6 +2629,88 @@ def Pareto(name, xm, alpha):
 
     return rv(name, ParetoDistribution, (xm, alpha))
 
+#-------------------------------------------------------------------------------
+# PERT distribution ----------------------------------------------------------
+
+
+class PERTDistribution(SingleContinuousDistribution):
+    _argnames = ('a', 'b', 'c')
+
+    @property
+    def set(self):
+        return Interval(self.a, self.c)
+
+    @staticmethod
+    def check(a, b, c):
+        _value_check((b > a, b < c),
+            "Parameter b must be in range (%s, %s). b = %s"%(a, c, b))
+
+    def pdf(self, x):
+        a, b, c = self.a, self.b, self.c
+        alpha = (4*b + c - 5*a)/(c - a)
+        beta = (5*c - a - 4*b)/(c - a)
+        num = (x - a)**(alpha - 1)*(c - x)**(beta - 1)
+        den = beta_fn(alpha, beta)*(c - a)**5
+        return num/den
+
+    def expectation(self, expr, var, **kwargs):
```

Comment:
Is the anticipated result not obtained for `E(X)` without this method?

Reply:
It is taking very long time.

Reply:
Can you open an issue for that? It could be an issue with integrals.

Reply:
@sidhantnagpal I have opened the issue: https://github.com/sympy/sympy/issues/16983